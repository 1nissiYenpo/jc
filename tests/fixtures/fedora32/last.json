[{"user": "kbrazil", "tty": "pts/0", "hostname": "192.168.71.1", "login": "Fri May 1 15:25", "logout": "gone - no logout"}, {"user": "kbrazil", "tty": "tty1", "hostname": null, "login": "Fri May 1 15:24", "logout": "gone - no logout"}, {"user": "reboot", "tty": "system boot", "hostname": "5.6.6-300.fc32.x", "login": "Fri May 1 15:24", "logout": "running"}, {"user": "kbrazil", "tty": "pts/0", "hostname": "192.168.71.1", "login": "Thu Apr 30 15:27", "logout": "15:22", "duration": "23:55"}, {"user": "kbrazil", "tty": "tty1", "hostname": null, "login": "Thu Apr 30 15:25", "logout": "down", "duration": "23:57"}, {"user": "reboot", "tty": "system boot", "hostname": "5.6.6-300.fc32.x", "login": "Thu Apr 30 15:22", "logout": "15:22", "duration": "1+00:00"}]
