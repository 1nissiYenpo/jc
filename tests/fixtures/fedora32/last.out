kbrazil  pts/0        192.168.71.1     Fri May  1 15:25    gone - no logout
kbrazil  tty1                          Fri May  1 15:24    gone - no logout
reboot   system boot  5.6.6-300.fc32.x Fri May  1 15:24   still running
kbrazil  pts/0        192.168.71.1     Thu Apr 30 15:27 - 15:22  (23:55)
kbrazil  tty1                          Thu Apr 30 15:25 - down   (23:57)
reboot   system boot  5.6.6-300.fc32.x Thu Apr 30 15:22 - 15:22 (1+00:00)

wtmp begins Thu Apr 30 15:22:02 2020
